haversine/haversine.py

```diff
@@ -24,15 +25,15 @@ class Unit(Enum):
     DEGREES = 'deg'
 
 
-class Direction(Enum):
+class Direction(float, Enum):
```

Comment:
Is this a breaking change ?

Reply:
It shouldn't be, it just makes the enum inherit from float so that it can be directly used as one. For example,
```
Direction.NORTH == 0.0
```
without going through `.value`